content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,269 @@
+      \                                      commit  /
+       \                                            /
+        \                                          /
+         \                                        /
+          \         v1.2.0               v1.2.1  /
+           ------------+---------------------+--/-----------> release-1.2
+                        \           backport ^ /
+                         \          commit  / /
+                          \                / /
+                           \              / /
+                            \            / /
+                             \          / /
+                              ------------------------------> master
+                                     ^   ^
+                            security |   | two new patch
+                               patch |   | releases
+                              merged       created immediately
+```
+
+<!-- Diagram source: https://textik.com/#7c4096204b3c0ad3 -->
+
+#### Critical bugs {#critical-bugs}
+
+**Critical bugs** include both regression bugs as well as upgrade bugs.
+
+Regressions are functionalities that did work in the previous release but
+no longer works. [#3393][] and [#2857][] are two examples of regressions.
+
+Upgrade bugs are bugs (often Helm-related) preventing users from upgrading
+to earlier final releases of cert-manager. [#3882][] and [#3644][] are
+example of such upgrade bugs.
+
+The fixes are back-ported; we then wait until the next final release for
+trigger a patch release. The patch release will be made for all supported
+release. Note that since we do only support the two last releases, patch
+releases are only made to the last release.
+
+In the below example, the release branches `release-1.2` and `release-1.3`
+are the two supported releases at the time of the release of 1.3.0, which
+means that the critical bug fix 1 and 2 will only be back-ported to
+`release-1.2` and not to `release-1.1`:
+
+```text
+   v1.2.1                                v1.2.2
+------+-------------------------------------+-----------> release-1.2
+       \      backport ^  backport ^        ^
+        \     commit  /   commit  /         | v1.2.2 is created
+         \           /           /          | along with v1.3.0
+          \         /           /
+           \       /           /
+            \     /           /          v1.3.0
+             --------master-----------------+-----------> release-1.3
+                 ^           ^               \
+                 |           |                \
+                 |           |                 \
+           critical      critical               \
+           bug fix 1     bug fix 2               \
+           merged to     merged to                \
+           master        master                    ------> master
+```
+
+<!-- Diagram source: https://textik.com/#7c4096204b3c0ad3 -->
+
+#### Long-standing bugs {#long-standing-bugs}
```

Comment:
suggestion: I feel like this section could be really short and simple. The diagram here feels
less important than other diagrams too, since it basically just shows cert-manager's
normal development cycle.

An example of text that could work here:

```text
Sometimes a bug exists for a long time, and may have known workarounds. [#3444][] is an
example of a long-standing bug.

Where we feel that back-porting would be difficult or might be a stability risk to
clusters running cert-manager, we'll make the fix in a major release but avoid
back-porting the fix.
```

Reply:
Yeah, you are right, my phrasing is convoluted for no reason... I also removed the (useless) diagram too.

Done in c636091.